index.js

```diff
@@ -6,7 +6,14 @@ const {
 
 module.exports = {
   convert: function (input, options, cb) {
-    const schema = new SchemaPack(input, options);
+    let schema;
```

Comment:
@diyaeninan Wouldn't same also be applied for other exposed functions `validate`, `getMetaData`` and `getOptions`?

Reply:
@VShingala The other exposed functions are :

- either not callback-based
- already handling thrown errors as expected
- do not throw errors explicitly from within the application logic

 hence this case should not arise, if it does it would be a bug